packages/storybook/stories/tabs.stories.tsx

```diff
@@ -86,6 +86,9 @@ export default TabsMeta;
 type Story = StoryObj<typeof Tabs>;
 
 export const Default: Story = {
+    args: {
+        global: false,
```

Comment:
Peut-être que `global = false` devrait être mis comme valeur par défaut à la prop dans le component à la place?

Reply:
Aucune idée ce que ça faisait là, mais why not!

Reply:
Ou juste l'enlever ici. Le component marche avec undefined aussi, c'est meme pas requis.